guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -5,8 +5,10 @@ You can use the Leapp tool to upgrade as well as to help detect and resolve issu
 
 .Prerequisites
 ifdef::satellite[]
-* Review Known Issues before you begin an upgrade.
+* Review known issues before you begin an upgrade.
 For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you use an HTTP proxy in your environment, configure the Subscription Manager to use the HTTP proxy for connection.
+For more information, see {RHELDocsBaseURL}9/html-single/upgrading_from_rhel_8_to_rhel_9/index#troubleshooting-resources_troubleshooting[Troubleshooting resources]. 
```

Comment:
I think this might be another case of the new RH docs portal misbehaving. The assembly (sometimes?) gets redirected to the first module, which might have caused the confusion about which link to add. This is the one that I believe we should use:

```suggestion
For more information, see {RHELDocsBaseURL}9/html-single/upgrading_from_rhel_8_to_rhel_9/index#troubleshooting_upgrading-from-rhel-8-to-rhel-9[Troubleshooting].
```

But since the link leads to a different docs set, you should probably also tell users where you are sending them too:


```suggestion
For more information, see {RHELDocsBaseURL}9/html-single/upgrading_from_rhel_8_to_rhel_9/index#troubleshooting_upgrading-from-rhel-8-to-rhel-9[Troubleshooting] in <guide name>.
```